vllm/worker/model_runner.py

```diff
@@ -631,6 +635,11 @@ def execute_model(
             model_executable = self.graph_runners[graph_batch_size]
         else:
             model_executable = self.model
+
```

Comment:
Kind of agree with what JZ suggested. Maybe we can have another "execute_model" method for embedding or put all embedding together inside a single if-else?

Reply:
Let's leave this part as it is for now. I think vLLM folks have different design ideas on enabling the embedding mode.